pkg/http/tripperware/tripperware.go

```diff
@@ -34,3 +35,9 @@ func DefaultWithTransport(transport *http.Transport, config *tls.Config) http.Ro
 	transport.TLSClientConfig = config
 	return &defaultTripper{Transport: transport}
 }
+
+func closeIfNotNil(r io.Closer) {
```

Comment:
Is there a specific reason why we are calling this `closeIfNotNil` instead of the `ensureClosed` that is used in the identical function in the `lbtransport`package?

I personally have a strong preference for `ensureClosed`.

Reply:
Mistake. Will get all to `ensureClosed`

Reply:
I don't think `ensureClosed` describes what this function is doing - I prefer `closeIfNotNil`, and even better would be a `closeBody(r *http.Request)` function.

Reply:
Talked to @Helcaraxan offline - moving to `closeIfNotNil`